src/notebook_unit1.ipynb

```diff
@@ -95,7 +95,7 @@
    "source": [
```

Comment:
<p>Needs to add .csv to .gitignore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>